<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>485</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New instrument</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QLabel" name="lblName">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="editID">
       <property name="enabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLineEdit" name="editValidateModule"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="lblId">
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QDateTimeEdit" name="dtLastCalibrated"/>
     </item>
     <item row="8" column="1">
      <widget class="QLineEdit" name="editFetchModule"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lblActive">
       <property name="text">
        <string>Active</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QLineEdit" name="editMatchExpression"/>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="lblFetchModule">
       <property name="text">
        <string>Fetch module</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="lblPrincipal">
       <property name="text">
        <string>Principal</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="lblValidateModule">
       <property name="text">
        <string>ValidateModule</string>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="QLabel" name="lblComment">
       <property name="text">
        <string>Comment</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QCheckBox" name="cbPrincipal">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QLineEdit" name="editStoreModule"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="editName"/>
     </item>
     <item row="12" column="1">
      <widget class="QLineEdit" name="editComment"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="lblChannelCount">
       <property name="text">
        <string>Channel count</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="editChannelCount"/>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="lblStoreModule">
       <property name="text">
        <string>Store module</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QCheckBox" name="cbActive">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="cboxStation"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lblStation">
       <property name="text">
        <string>Station</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="lblLastCalibibrated">
       <property name="text">
        <string>Last calibrated</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="lblMatchExpression">
       <property name="text">
        <string>Match Expression</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="editModel"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="lblModel">
       <property name="text">
        <string>Model</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
